### PR TITLE
Update ios_14.md

### PR DESCRIPTION
**Pull request by davidbielik**

### CODE REVIEW

File: _docs/_developer_guide/platform_integration_guides/ios/ios_14.md [hunk 1]
```diff
@@ -11,17 +11,16 @@ This guide describes Braze-related changes introduced in iOS 14 and the required
...
 {% endalert %}
 
 #### Summary of iOS 14 breaking changes
 
 - Geofences are [no longer supported by iOS][4] for users who choose the new  _approximate location_ permission.
-- Customers using the "Last Known Location" targeting features are required to upgrade their Braze iOS SDK to at least v3.26.1 for compatibility with _approximate location_ permission.
-- IDFA collection will [soon require](https://developer.apple.com/news/?id=hx9s63c5&1599152522) a permission prompt. Once Apple begins to enforce this change, apps must update to use the new IDFA collection methods.
+- Use of the "Last Known Location" targeting features will require an upgrade to Braze iOS SDK v3.26.1 for compatibility with _approximate location_ permission.
+- IDFA collection will [soon require](https://developer.apple.com/news/?id=hx9s63c5&1599152522) a permission prompt. Once Apple begins to enforce this change later in 2021, apps must update to use the new AppTrackingTransparency Framework.
```
Comment (Bucimis) [inline _docs/_developer_guide/platform_integration_guides/ios/ios_14.md:21]: Wherever we say AppTrackingTransparency  first we should link to https://developer.apple.com/documentation/apptrackingtransparency

**Review by Bucimis — DISMISSED** [review]

**Review by timothy-kim — DISMISSED** [review]
👌 lmk when you want this to be merged

**Review by KellieHawks — APPROVED** [review]
lgtm!

**Review by lowip — APPROVED** [review]
lgtm!

**Review by timothy-kim — APPROVED** [review]

**Pull request merged**